src/scss/common/careers.scss

```diff
@@ -15,6 +15,16 @@
   }
 }
 
+.career-title {
+  background-color: black;
+  color: white;
+  padding-top: 0.5rem;
+  padding-bottom: 0.5rem;
+  padding-left: 1rem;
+  padding-right: 1rem;
+  text-align: center;
+}
```

Comment:
It might be worth adding a `margin-bottom: 0` as there's been an extra margin added by using h4 instead of button. 

Also padding can be combined into shorthand rather than 4 lines. I'd maybe change the left/right padding to 2rem instead of 1rem, like this: `padding: 0.5rem 2rem;`

And add `border-radius: .25rem;` also for the rounded edges.

Reply:
👍